src/czbenchmarks/datasets/single_cell_perturbation.py

```diff
@@ -91,13 +91,13 @@ def __init__(
         organism: Organism,
         condition_key: str = "condition",
         control_name: str = "ctrl",
+        condition_control_sep: str = "_",
         de_gene_col: str = "gene",
-        deg_test_name: str = "wilcoxon",
+        # FIXME MICHELLE should these be moved to load_data?
```

Comment:
I don't think so, but explain your reasoning? E.g., why only the ones below and not the ones above, when all of these are used in the _create_adata() func?